packages/filesystem/src/node/node-filesystem.spec.ts

```diff
@@ -5,36 +5,27 @@
  * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
  */
 
+import * as os from 'os';
+import * as temp from 'temp';
 import * as chai from 'chai';
+import * as fs from 'fs-extra';
 import * as assert from 'assert';
 import * as chaiAsPromised from 'chai-as-promised';
-import * as fs from 'fs-extra';
-import * as os from 'os';
 import URI from "@theia/core/lib/common/uri";
-import { Logger } from "@theia/core/lib/common";
 import { FileUri } from "@theia/core/lib/node";
 import { PreferenceService, PreferenceServer, PreferenceClient } from "@theia/preferences-api";
 import { FileSystem } from "../common/filesystem";
 import { FileSystemWatcher, FileChange, FileChangeType, createFileSystemPreferences } from '../common';
 import { FileSystemNode } from "./node-filesystem";
-import { ChokidarFileSystemWatcherServer } from './chokidar-filesystem-watcher';
+import { NsfwFileSystemWatcherServer } from './nsfw-watcher/nsfw-filesystem-watcher';
+
+// tslint:disable:no-unused-expression
```

Comment:
Since you have already cleaned up the test setup, perhaps, you could get rid of `chai-as-promised` too, and use `async`, can you? (First, let's have green tests on the CI.)

Reply:
Follow-up: https://github.com/theia-ide/theia/issues/928